examples/ddi_seal/dataloader.py

```diff
@@ -0,0 +1,115 @@
+import torch
+from torch_geometric.data import InMemoryDataset, Dataset
+from utils import get_pos_neg_edges, extract_enclosing_subgraphs, k_hop_subgraph, construct_pyg_graph
+from torch_sparse import coalesce
+import scipy.sparse as ssp
+
+
+class SEALDataset(InMemoryDataset):
```

Comment:
Is the dataloader example-specific rather than generic? If it can be reused by others, it will be good to be put in `kale.loaddata`. See examples of digit/cifar/video loader there.

Reply:
Also, did you build this from some existing code? If yes, please put the link here and specify in the docstrings.